README.md

```diff
@@ -25,7 +25,7 @@ Sign up to [get started for free](https://app.revenuecat.com/signup).
 *RevenueCat* is the client for the [RevenueCat](https://www.revenuecat.com/) subscription and purchase tracking system. It's 100% `Swift` and compatible with `Objective-C`.
 
 ## Migrating from Purchases v3
-- See our [Migration guide](https://revenuecat-docs.netlify.app/documentation/revenuecat/v4_api_migration_guide)
+- See our [Migration guide](https://revenuecat.github.io/purchases-ios-docs/4.17.8/documentation/revenuecat/v4_api_migration_guide)
```

Comment:
Hmm if we update those docs in later versions (unlikely but possible), we would need to remember to update this... I wonder if there is a way to point to latest docs instead?

Reply:
yeah, good point. I'll address that in a follow-up PR, but we should be able to merge this for now since it replaces a broken link for a functioning one, even if it can get out of sync

Reply:
related: https://github.com/RevenueCat/purchases-ios-docs/pull/3